.github/build_real.sh

```diff
@@ -7,4 +7,11 @@ if [[ $IMAGE == "private" ]]; then
     cp -r $HOME/SNOPT/* pyoptsparse/pySNOPT/source
 fi
 
+# temporarily disable pip constraints file for build-time dependencies
+mv ~/.config/pip/constraints.txt ~/.config/pip/constraints.txt.bkup
```

Comment:
FYI @eirikurj this was the only way I could to bypass pip constraint files -- I'm surprised they're still respected in the build env, is that even PEP517 compliant?

Reply:
Dont recall exactly, but I think they operate outside of PEP 517, i.e., you can still use constraint files when installing packages built using PEP517 compliant build systems.